apply.js

```diff
@@ -1,24 +1,55 @@
-const jsPatches = ["displayFullName"];
-const cssPatches = [
-  "hideWCAG",
-  "alignDetailedGradesButton",
-  "hideTutorsFromBoard",
+/**
```

Comment:
Do we really need jsdoc type definitions in only one place in a project 🤷‍♂️

Reply:
I was thinking about future contributors who will less understand this code, but you are right